study.md

```diff
@@ -21,18 +21,19 @@ There is no requirement to follow any links from these pages.
 After doing all the required readings please tell us anything you were confused about.
 
 ```md
-<!-- your answer here -->
+<!-- I am still hazzy on the specifics on functions, and the idea of call back functions. Also how they all come togther. I think I understand most of this conceptually, but putting it together is where I get lost. Especially when building functions on top of and intereacting with other functions. -->
 ```
 
 Are there any nagging questions relating to functions that you have?
 
 ```md
-<!-- your answer here -->
+<!-- I am curious about how callback comes together, and interacts amougnst other code. Especially the single line functions. I am also unsure about object.method? if thats what it actually is. -->
```

Comment:
We'll be going over array methods this coming week and many of them have callback functions as parameters.  That should help you understand the purpose of callback functions.